lib/workers/repository/init/types.ts

```diff
@@ -5,6 +5,7 @@ export interface RepoConfigError {
 
 export interface RepoFileConfig {
   configFileName?: string;
+  configFileRaw?: string;
```

Comment:
```suggestion
  configFileRaw: string | null;
```

property is now always there, but can be null

Reply:
this breaks too many things, the returned object doesnt always contain the raw data, not when there are validation errors and when using legacy `package.json` as a config file.

`string | undefined` is preferred in this case. this object is also used in some other places, which means that if we go for `string | null`, this for example

https://github.com/renovatebot/renovate/blob/8586f463ea0b182fe274c2d4c50a79792c860390/lib/workers/repository/init/merge.ts#L172-L176

will have to become 

```ts
   let repoConfig: RepoFileConfig = {configFileRaw: null}; 
```
change which i dont like too much

the same will have to be done in quite a few other tests as well

Reply:
updated my suggestion